osu.Game/Screens/Select/PlaySongSelect.cs

```diff
@@ -189,9 +189,13 @@ private void start()
         }
 
         private ScheduledDelegate filterTask;
+        private string previousFilter = null;
 
         private void filterChanged()
         {
+            if (previousFilter == filter.Search)

```

Comment:
did you mean to use this?

![image](https://cloud.githubusercontent.com/assets/191335/22490860/04d5a970-e863-11e6-9b89-97b315f09347.png)